src/services/Models.service.tsx

```diff
@@ -374,6 +506,63 @@ export const expressions = {
 /**
  * Animates avatars facial expressions when morphTargets=ARKit,Eyes Extra is provided with the avatar.
  */
+export const useIdleExpressionScene = (expression: keyof typeof expressions, scene: Group<Object3DEventMap>) => {
```

Comment:
The timeout had potential to leak as the headMesh may still have been referenced when a timeout was active.